library/amazonlinux

```diff
@@ -1,15 +1,16 @@
 Maintainers: Amazon Linux Team <amazon-linux-ami@amazon.com> (@aws),
              Iliana Weller (@ilianaw),
-             Praveen K Paladugu (@praveen-pk)
+             Praveen K Paladugu (@praveen-pk),
+	     Eric Warehime (@Eric-Warehime)
```

Comment:
This needs to be spaces rather than a tab.

Reply:
Fixing it now 💃